cli/app.go

```diff
@@ -2755,106 +2755,6 @@ This won't work unless you have an existing installation of our GitHub app on yo
 				},
 			},
 		},
-		{
-			Name:            "auth-app",
```

Comment:
Question: anything special i should do when removing this command? is this an ok breaking change? dont think anyone was using these since they were so simple. ill make sure docs are updated too

Reply:
> dont think anyone was using these since they feature complete.

I'm not sure I understand this sentence, do you mind rephrasing?

Re: the breaking change, I guess I want to understand a little bit more what the impetus for this is. It looks to me like the proto `AuthApp` methods still exist... is there a plan to delete those entirely? If not, how do we intend for them to still be used, and why are we wanting to delete them from the CLI but not elsewhere?

Reply:
Got it, thanks for clarifying! In that case, this looks reasonable to me :)